js/chkForm.js

```diff
@@ -0,0 +1,22 @@
+$(document).ready(function(){
+
+  chkForm();
+
+});
+
+$(document).keyup(function(){
+
+  chkForm();
+
+});
+
+function chkForm() {
+  if($("#contact_name").val() == "" || $("#contact_mail").val() == "" || $("#contact_message").val() == ""){
```

Comment:
__[jshint]__
*  Use '===' to compare with ''.


<!-- Comment by [SideCI](https://sideci.com) -->